netebpfext/net_ebpf_ext_program_info.h

```diff
@@ -166,8 +167,21 @@ static const ebpf_program_type_descriptor_t _ebpf_sock_ops_program_type_descript
     EBPF_PROGRAM_TYPE_SOCK_OPS_GUID,
     BPF_PROG_TYPE_SOCK_OPS,
     0};
+// SOCK_OPS global helper function prototypes.
+static const ebpf_helper_function_prototype_t _ebpf_sock_ops_global_helper_function_prototype[] = {
+    {.header = EBPF_HELPER_FUNCTION_PROTOTYPE_HEADER,
+     .helper_id = BPF_FUNC_get_current_pid_tgid,
```

Comment:
I think we cannot change the name of this helper function. There are some programs that already have dependency on the older helper function.

Reply:
@saxena-anurag Can we ask for a recompile?  (so `#define` old in terms of new)
If not, can we mark older as deprecated and remove in a later release?

Reply:
Agree. It seems that change has been reverted now, and the older helper has a comment that this is only for legacy programs.